docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -14,19 +21,29 @@ All the steps in the graphics pipeline go from top to bottom and are shown below
 To draw most of the things throughout this series we'll mostly need these stages: Input Assembler, Vertex Shader and the Pixel Shader, Output Merger.
 
-The Vertex and Pixel shaders are fully programmable, and we'll write a very basic program for them.
+The Vertex and Pixel shaders are fully programmable and we'll write a very basic program for them.
 
-The other two stages are not programmable, but they are fairly easy to understand and configure, the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive of our choice, which is of course, triangles in our case, and sending this processed output to the Vertex Shader. The **Output Merger** instead is responsible for combining the values written by the pixel shader, may that be depth, color or other things, into the one or more render targets that we provide to the OM, we only have one render target for now.
+The other two stages are not programmable but they are fairly easy to understand and configure:
 
-### Vertex Shader
+- the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive topology of our choice, which in our case is a form of triangles, and sending this processed output to the Vertex Shader to process. 
+- The **Output Merger** is responsible for combining the values written by the pixel shader, may that be depth, color or other things, into the one or more render targets that we provide to the OM, we only have one render target for now.
 
-The **Vertex Shader** is the stage where our vertices are processed however we want, although we don't do much processing here, and in the end they're transformed to [screen-space coordinates](link to the coordinate system chapter?)
+The **Vertex Shader** is the stage where our vertices are processed however 
+we want, although we don't do much processing here, and in the end they're 
+transformed to [screen-space coordinates](link to the coordinate system chapter?)
 
-The vertices are usually read from a **Vertex Buffer** which are laid out in a particular way. The vertex shader will be run however many times we tell it to run, which is specified in the first parameter of `ID3D11DeviceContext::Draw()` (more on this later), for instance if we call `Draw(3, 0)`, the vertex shader will run 3 times.
+The vertices are usually read from a **Vertex Buffer** which are laid out in 
+a particular way. The vertex shader will be run however many times we tell 
+it to run, which is specified in the first parameter of `ID3D11DeviceContext::Draw()` 
+(more on this later), for instance if we call `Draw(3, 0)`, the vertex shader will 
+run 3 times.
 
-Since we only want to draw a triangle, we don't need to do much processing, we can just provide the input vertices as the output.
+Since we only want to draw a triangle, we don't need to do much processing, 
```

Comment:
> Since we only want to draw a triangle, we don't need to do much processing,

overly simplified in my opinion, this gives the reader the notion that each triangle only needs direct forwarding to the PS, it would be more correct to note that "Since the vertex positions we supply are already in NDC space we don't need to transform/process them for the rasterisation stage, so we can just provide the input vertices as the output"

Reply:
I am confused at your comment- I will specify a bit more though since I think that helps with your comment?